source/dub/project.d

```diff
@@ -403,6 +410,7 @@ class Project {
 			}
 		}
 		collectDependenciesRec(m_rootPackage);
+		m_missingDependencies = m_missingDependencies.sort().array;
```

Comment:
`m_missingDependencies.sort();` will already sort the source array, you don't have to allocate a new one with `.array`

Reply:
```suggestion
		m_missingDependencies.sort();
```

Reply:
fixed